<!DOCTYPE html>
<html lang="en">
  <head>
  <!-- put your analytics (e.g. Google Analytics) tracking code here -->

  










<title>Research | EpiFluid Lab</title>

<meta name="viewport" content="width=device-width, initial-scale=1" />

<!-- https://metatags.io/ -->
<meta name="title" content="Research | EpiFluid Lab" />
<meta name="description" content="We are epigenomics and computational biology lab within the Division of Human Genetics and Biomedical Informatics in the Cincinnati Children’s Hospital Medical Center (CCHMC) and also part of the Department of Pediatrics in University of Cincinnati College of Medicine." />
<meta property="og:type" content="website" />
<meta property="og:url" content="" />
<meta property="og:title" content="Research | EpiFluid Lab" />
<meta property="og:description" content="We are epigenomics and computational biology lab within the Division of Human Genetics and Biomedical Informatics in the Cincinnati Children’s Hospital Medical Center (CCHMC) and also part of the Department of Pediatrics in University of Cincinnati College of Medicine." />
<meta property="twitter:card" content="summary_large_image" />
<meta property="twitter:url" content="" />
<meta property="twitter:title" content="Research | EpiFluid Lab" />
<meta property="twitter:description" content="We are epigenomics and computational biology lab within the Division of Human Genetics and Biomedical Informatics in the Cincinnati Children’s Hospital Medical Center (CCHMC) and also part of the Department of Pediatrics in University of Cincinnati College of Medicine." />

<!-- RSS -->
<link
  rel="alternate"
  type="application/rss+xml"
  href="/feed.xml"
/>

<!-- https://developers.google.com/web/fundamentals/native-hardware/click-to-call#disable_auto-detection_when_necessary -->
<meta name="format-detection" content="telephone=no" />

  <!-- https://realfavicongenerator.net/ -->
<link
  rel="apple-touch-icon"
  sizes="180x180"
  href="/favicons/ios.png"
/>
<link
  rel="icon"
  type="image/png"
  sizes="512x512"
  href="/favicons/favicon.png"
/>
<link rel="manifest" href="/favicons/site.webmanifest" />

<!-- https://metatags.io/ -->
<meta
  property="og:image"
  content="/favicons/share-thumbnail.jpg"
/>
<meta
  property="twitter:image"
  content="/favicons/share-thumbnail.jpg"
/>

  <!-- Google Fonts -->
<link rel="preconnect" href="https://fonts.gstatic.com" />
<link
  href="https://fonts.googleapis.com/css2?family=Cousine:ital,wght@0,400;0,700;1,400;1,700&family=Lora:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500;1,600;1,700&family=Poppins:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500;1,600;1,700&display=swap"
  rel="stylesheet"
/>

<!-- Font Awesome icons (load asynchronously due to size) -->
<link
  href="https://use.fontawesome.com/releases/v6.0.0/css/all.css"
  rel="preload"
  as="style"
  onload="this.onload = null; this.rel = 'stylesheet';"
/>
<noscript>
  <link
    href="https://use.fontawesome.com/releases/v6.0.0/css/all.css"
    rel="stylesheet"
  />
</noscript>

  <!-- site-wide css styles -->
      <link href="/css/all.css" rel="stylesheet" />
      <link href="/css/anchor.css" rel="stylesheet" />
      <link href="/css/background.css" rel="stylesheet" />
      <link href="/css/banner.css" rel="stylesheet" />
      <link href="/css/button.css" rel="stylesheet" />
      <link href="/css/card.css" rel="stylesheet" />
      <link href="/css/citation.css" rel="stylesheet" />
      <link href="/css/code.css" rel="stylesheet" />
      <link href="/css/feature.css" rel="stylesheet" />
      <link href="/css/figure.css" rel="stylesheet" />
      <link href="/css/footer.css" rel="stylesheet" />
      <link href="/css/form.css" rel="stylesheet" />
      <link href="/css/gallery.css" rel="stylesheet" />
      <link href="/css/header.css" rel="stylesheet" />
      <link href="/css/heading.css" rel="stylesheet" />
      <link href="/css/highlight.css" rel="stylesheet" />
      <link href="/css/icon.css" rel="stylesheet" />
      <link href="/css/image.css" rel="stylesheet" />
      <link href="/css/input.css" rel="stylesheet" />
      <link href="/css/link.css" rel="stylesheet" />
      <link href="/css/list.css" rel="stylesheet" />
      <link href="/css/paragraph.css" rel="stylesheet" />
      <link href="/css/portrait.css" rel="stylesheet" />
      <link href="/css/post-excerpt.css" rel="stylesheet" />
      <link href="/css/post-info.css" rel="stylesheet" />
      <link href="/css/post-nav.css" rel="stylesheet" />
      <link href="/css/quote.css" rel="stylesheet" />
      <link href="/css/role.css" rel="stylesheet" />
      <link href="/css/rule.css" rel="stylesheet" />
      <link href="/css/search-box.css" rel="stylesheet" />
      <link href="/css/search-info.css" rel="stylesheet" />
      <link href="/css/section.css" rel="stylesheet" />
      <link href="/css/table.css" rel="stylesheet" />
      <link href="/css/tags.css" rel="stylesheet" />
      <link href="/css/tooltip.css" rel="stylesheet" />
      <link href="/css/two-col.css" rel="stylesheet" />
      <link href="/css/util.css" rel="stylesheet" />
  <!-- third party scripts -->
<script src="https://unpkg.com/@popperjs/core@2" defer></script>
<script src="https://unpkg.com/tippy.js@6" defer></script>
<script
  src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js"
  defer
></script>

<!-- site-wide javascript scripts -->
<script src="/js/anchors.js" defer></script>
  <script src="/js/fetch-tags.js" defer></script>
  <script src="/js/search.js" defer></script>
  <script src="/js/site-search.js" defer></script>
  <script src="/js/tooltips.js" defer></script>
  <script src="/js/true-hide.js" defer></script>
  <script src="/js/util.js" defer></script>
  

</head>

  <body>
    <header   data-dark="true">
  <a
    href="/"
    class="logo_row"
    data-tooltip="Home"
  >
    <img src="/images/logo.epifluid.png" class="logo" alt="Logo image" />
    
      <span class="logo_text">
        <span class="logo_title">EpiFluid Lab</span>
        
          <span class="logo_subtitle">The dark matter of the genome -- Understand the non-coding genetic variants by multi-omics inside and outside the cells.</span>
        
      </span>
    
  </a>
  <nav class="nav_row"><a class="link" href="/research/" data-tooltip="Published works"  data-text="true"  data-style=""><span>Research</span></a><a class="link" href="/team/" data-tooltip="About our team"  data-text="true"  data-style=""><span>Team</span></a><a class="link" href="/publications/" data-tooltip="Published works"  data-text="true"  data-style=""><span>Publications</span></a><a class="link" href="/tools/" data-tooltip="Software, datasets, and more"  data-text="true"  data-style=""><span>Software</span></a><a class="link" href="/news/" data-tooltip="Published works"  data-text="true"  data-style=""><span>News</span></a><a class="link" href="/openings/" data-tooltip="Published works"  data-text="true"  data-style=""><span>Join us</span></a><a class="link" href="/group_photo/" data-tooltip="group photos"  data-text="true"  data-style=""><span>Group photos</span></a><a class="link" href="/contact/" data-tooltip="Email, address, and location"  data-text="true"  data-style=""><span>Contact</span></a></nav>
</header>

    <main>
      <section
      class="background"
      
      
      
    ><h1 id="research"><i class="fas fa-microscope"></i>Research</h1>

<p>Our long-term research interest is to decode the human genome. The recent research focus of my lab is on developing and applying computational and high-throughput experimental methods in epigenomics to understand the gene regulatory roles of non-coding genetic variants in different pathological conditions, including cancers and neurodegenerative diseases. Specifically, we focused on the computational method development for circulating cell-free DNA fragmentation and new biotechnology development for the joint profiling of multi-omics within the same single cells, which will eventually enable biomarker discovery for the early diagnosis and prognosis of many complex diseases.</p>

<h1 id="current-projects"><i class="fas fa-microscope"></i>Current Projects</h1>

<h2 id="non-invasive-monitor-the-in-vivo-epigenomes-by-circulating-cell-free-dna">Non-invasive monitor the <em>in vivo</em> epigenomes by circulating cell-free DNA</h2>
<p>Circulating cell-free DNA (cfDNA) is released into the peripheral blood after cellular death and recycled every few mins up to 12 hours. The collection of cfDNA fragments represents a real-time <em>in vivo</em> snapshot of the genome from cells that contribute to cfDNA. CfDNA is not randomly fragmented into small pieces and recently associated with the cellular epigenomes, suggesting the possibility of computationally inferring the cellular epigenomes from cfDNA fragmentation. However, the computational tools are still not available to comprehensively infer the cellular epigenomes from cfDNA whole-genome sequencing (WGS). Millions of low-coverage and high-coverage cfDNA WGS are generated every year. Given the potential to leverage these cfDNA WGS datasets to advance the understanding of the epigenomes and thus the function of non-coding regulatory elements, we are developing a set of computational methods to reconstruct the multi-dimensional epigenomes from a single cfDNA WGS.</p>

<figure class="figure">
  <a class="figure_image" style="">
    <img src="/images/cfdna_epi.png" alt="Utilize cfDNA fragmentation pattern and AI to monitor the gene regulation within the cells during the disease initiation and progression." title="Utilize cfDNA fragmentation pattern and AI to monitor the gene regulation within the cells during the disease initiation and progression." style="width: 1000px;" loading="lazy" />
  </a><figcaption class="figure_caption">Utilize cfDNA fragmentation pattern and AI to monitor the gene regulation within the cells during the disease initiation and progression.
</figcaption></figure>

<h2 id="develop-methods-to-jointly-profile-multi-omics-in-the-same-single-cells">Develop methods to jointly profile multi-omics in the same single cells</h2>
<p>Epigenetic modifications, including DNA methylation, histone modifications, and three-dimensional (3D) genome topology, combine with genetic content to determine the mammalian transcriptional factor (TF) binding and, thus, gene regulation. Gene activation or repression potential, however, cannot be entirely predicted by looking at a single molecular measurement. Accurate predictive models require multiple molecular measurements simultaneously. At present, we are limited by the number of simultaneous measurements that we can perform in a single cell. In addition, the interactions between different epigenetic marks and their effects on gene expression are currently studied either in homogenous cultured cells or bulk tissues that average the readout. The study of interactions between different cell-type-specific epigenetic marks and gene expression in heterogeneous tissues at the single cell level is still in its infancy. We and others developed several technologies to simultaneously capture multiple molecular measurements in the same assay or single-cells. We continue to develope more high-throughput mutli-omic technologies to comprehensively dissect the regulatory roles of the non-coding regulatory elements in the genome.</p>

<figure class="figure">
  <a class="figure_image" style="">
    <img src="/images/singlecell_multi.png" alt="Develop single-cell multi-omics technologies to understand gene regulation" title="Develop single-cell multi-omics technologies to understand gene regulation" style="width: 800px;" loading="lazy" />
  </a><figcaption class="figure_caption">Develop single-cell multi-omics technologies to understand gene regulation
</figcaption></figure>

<h2 id="epigenomics-in-liquid-biopsy-for-the-early-diagnosis-and-prognosis-of-diseases">Epigenomics in liquid biopsy for the early diagnosis and prognosis of diseases</h2>
<p>Routine screening in the average-risk population provides the opportunity to detect cancer early and significantly reduce morbidity and mortality in cancer. Early identification of lethal cancer versus non-lethal diseases will minimize overdiagnosis. Both medical needs require non-invasive biomarkers with ultra-high specificity (&gt;99%) and high sensitivity at the same time, which, however, are still not available in the clinic. Recent advances in circulating cell-free DNA (cfDNA) suggested a promising non-invasive approach for cancer diagnosis and prognosis by using the tumor-specific genetic and epigenetic alterations. However, there is still a huge gap to limit its clinical application for cancer screening. We are utilizing the epigenomic signals from circulating cfDNA for the early diagnosis and prognosis of cancers. Moreover, we are expanding the studies in cancer to many other complex diseases.</p>

<figure class="figure">
  <a class="figure_image" style="">
    <img src="/images/diagnosis.png" alt="Epigenomics in liquid biopsy for disease diagnosis" title="Epigenomics in liquid biopsy for disease diagnosis" style="width: 500px;" loading="lazy" />
  </a><figcaption class="figure_caption">Epigenomics in liquid biopsy for disease diagnosis
</figcaption></figure>

<h1 id="funding">Funding</h1>

<p>Our work is currently (and was previously) supported by <strong>NIH</strong> (R35GM147283, R56HG012360), <strong>NSF</strong> (ACCESS and XSEDE Resource Allocation Service), <strong>Bill &amp; Melinda Gates Foundation</strong> (MOMI Ideas Award), and <strong>Northwestern University</strong> (Yaping Liu’s start-up).</p>
</section>
    </main>
    <footer   data-dark="true">

  <!--
  <div class="footer_row">
    Extra details like contact info or address
  </div>
  -->

  <div class="footer_row"><a class="link" href="mailto:Yaping.Liu@cchmc.org" data-tooltip="Email" data-icon="true"   data-style=""><i class="fas fa-envelope fa-lg fa-fw"></i></a><a class="link" href="https://scholar.google.com/citations?hl=en&user=Yaping Liu" data-tooltip="Google Scholar" data-icon="true"   data-style=""><i class="fab fa-google fa-lg fa-fw"></i></a><a class="link" href="https://github.com/epifluidlab" data-tooltip="GitHub" data-icon="true"   data-style=""><i class="fab fa-github fa-lg fa-fw"></i></a><a class="link" href="https://twitter.com/yapingliu" data-tooltip="Twitter" data-icon="true"   data-style=""><i class="fab fa-twitter fa-lg fa-fw"></i></a><a class="link" href="https://orcid.org/0000-0003-4165-7693" data-tooltip="ORCID" data-icon="true"   data-style=""><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 256 256" height="1.5em">
  <path
    fill="currentColor"
    d="M256,128c0,70.7-57.3,128-128,128C57.3,256,0,198.7,0,128C0,57.3,57.3,0,128,0C198.7,0,256,57.3,256,128z"
  />
  <g>
    <path fill="#ffffff" d="M86.3,186.2H70.9V79.1h15.4v48.4V186.2z" />
    <path
      fill="#ffffff"
      d="M108.9,79.1h41.6c39.6,0,57,28.3,57,53.6c0,27.5-21.5,53.6-56.8,53.6h-41.8V79.1z M124.3,172.4h24.5c34.9,0,42.9-26.5,42.9-39.7c0-21.5-13.7-39.7-43.7-39.7h-23.7V172.4z"
    />
    <path
      fill="#ffffff"
      d="M88.7,56.8c0,5.5-4.5,10.1-10.1,10.1c-5.6,0-10.1-4.6-10.1-10.1c0-5.6,4.5-10.1,10.1-10.1C84.2,46.7,88.7,51.3,88.7,56.8z"
    />
  </g>
</svg>
</a><a class="link" href="https://www.linkedin.com/in/yaping-liu-b746541a" data-tooltip="LinkedIn" data-icon="true"   data-style=""><i class="fab fa-linkedin fa-lg fa-fw"></i></a></div>

  <div class="footer_row">
    <span>
      &copy; 2023 EpiFluid Lab
      &nbsp;
    </span>
  </div>

</footer>

  </body>
</html>
